.travis.yml

```diff
@@ -17,30 +17,59 @@ env:
     #   See https://docs.travis-ci.com/api#external-apis.
     - secure: "qQ3vMSXUybUyKljJjgcVT9YaWcZ0PkdLZSFj6BVuo82SWO7e4L+hC8pqsa1Sm/EjyetVwfKk5lkvruPpro++8hDPLq3ahNOMmm4axfcWvACpW+acD/VCA5D8Dxz2ohrvpwsqo7BmIV6OUtIO5JeePHP1SqaIdVZFJ0pIAZToEWQ="
     - CONDA_ROOT_PYTHON_VERSION: "2.7"
-    - NUMPY_VERSION=1.11.1
-    - PANDAS_VERSION=0.18.1
-    - SCIPY_VERSION=0.17.1
+  matrix:
+    - PANDAS_18=1
+    - PANDAS_22=1
```

Comment:
What does the syntax mean here? Does`PANDAS_22` to `1`  generate an environment variable?

Reply:
Yup, exactly. It sets that env var.